cypress/e2e/7-workflow-actions.cy.ts

```diff
@@ -108,6 +108,23 @@ describe('Workflow Actions', () => {
 		cy.wait('@saveWorkflow');
 		cy.wrap(null).then(() => expect(interceptCalledCount).to.eq(1));
 	});
+
+	it('should not save workflow twice when save is in progress', () => {
+		// This happens when users click save button from workflow name input
+		// In this case blur on the input saves the workflow and then click on the button saves it again
+		WorkflowPage.actions.visit();
+		WorkflowPage.getters.workflowNameInput().invoke('val').then((oldName) => {
+			WorkflowPage.getters.workflowNameInputContainer().click();
+			WorkflowPage.getters.workflowNameInput().type('{selectall}');
```

Comment:
Is this some Cypress magic to select all the text?

Reply:
@tomi Yeah, there is a few of those handy options. You can see them [here](https://docs.cypress.io/api/commands/type#Arguments).